terraform/modules/elasticache/security_groups.tf

```diff
@@ -0,0 +1,25 @@
+resource "aws_security_group" "redis" {
+  name        = "${var.prefix}-redis-security-group"
+  description = "Redis security group"
+  vpc_id      = var.vpc_id
+
+  ingress {

```

Comment:
It looks like the terraform docs have a note suggesting that people use `aws_vpc_security_group_ingress_rule` (and `..._egress_rule`) for new resources (sorry, I didn't end up looking at the docs for previous reviews). It's unclear whether we'll come up against they problem they mention, but it might be worth updating everywhere while that's still easy to change. 

(Although I'm saying that expecting it to be a quick thing to change, so let me know if that's wrong because we could decide to leave it until we actually hit a problem)

Reply:
Ah yes good spot! It doesn't look too bad to change (I've worked out the syntax already), but can't apply it because it doesn't let you get away with providing a blank security group source id. We don't have one until we make our ECS security group in a later PR, so I'd prefer to make the change there if that's ok (for RDS, DB and ECS sg's)?